i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/map-web-update.md

```diff
@@ -69,8 +95,16 @@ sudo yum update centreon-map-engine centreon-map-web-client
 
 3. Videz le cache de votre navigateur.
 
-4. Redémarrez le service **centreon-map-engine** en exécutant la commande suivante :
+4. Finalisez la mise à jour du module et du widget dans l'interface Centreon **Administration > Extensions > Gestionnaire**.
+
+5. Redémarrez le service **centreon-map-engine** en exécutant la commande suivante :
  
   ```shell
   sudo systemctl start centreon-map-engine
   ```
+
+6. Si vous utilisez MAP Legacy, entrez cette commande sur le serveur MAP pour reporter une éventuelle personnalisation.
+
+ ```shell
+ diff -u /etc/centreon-studio/centreon-map.conf /etc/centreon-studio/centreon-map.conf.rpmsave
```

Comment:
This command only works for EL7/EL8/EL9 but not on Debian.
Debian will not create .rpmsave files
Also, if we update Centreon Map, we don't care about /etc/centreon-studio/ folder because it's related to centreon Map Legacy and the update will not modify these files